src/general/Mutations.cs

```diff
@@ -235,7 +235,16 @@ private void MutateBehaviour(MicrobeSpecies parent, MicrobeSpecies mutated)
                     }
                     else
                     {
-                        AddNewOrganelle(mutatedOrganelles, parentOrganelles.Organelles.Random(random).Definition);
+                        // Duplicate an existing organelle, but only if there are any organelles where that is legal
+                        var organellesThatCanBeDuplicated = parentOrganelles.Organelles.Where(organelle => !organelle.Definition.Unique);
+                        if (organellesThatCanBeDuplicated.Any())
```

Comment:
I kinda would like to have this be instead so that first ToList() is called, on that then Count > 0 is checked, and then the random is retrieved. I suggest this because I think .Any will first enumerate at least one item, and then the ToList call will re-enumerate that, which seems to be exactly what CI is complaining here about.